app/views/placeholder/landing.html.erb

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link href='https://fonts.googleapis.com/css?family=Roboto Slab' rel='stylesheet'>
+    <style>
+    html, body {
+      background-image: url('<%= asset_path('prisms.png') %>'); 
+      font-family: 'Roboto Slab';
+      height:100%; /* important to vertically align the container */
+      margin:0;
+      padding:0;
+    }
+    .vertical-center {
+      min-height: 100%;  /* Fallback for browsers do NOT support vh unit */
+      min-height: 100vh; /* These two lines are counted as one 🙂       */
+      display: flex;
+      align-items: center;
+    }
+    </style>
+    <title>Hello, world!</title>
+  </head>
+  <body>
+  
+  
+
+
+<div class="vertical-center" >
+  <div class="container rounded" style="background-color: #FFFFFF; max-width:1000px" >
+    <div class="row">
+      <div class="col-sm-12" style="text-align:center">
+          <%=image_tag("LEAD logo.png", :align=> "middle", :style => "width:300px")%>

```

Comment:
Maybe some padding on top of the bear and under the login button.

Reply:
I think we should discuss how we're gonna be figuring out and deciding design feedback before we start our next iteration.